### PR TITLE
Remove `updateAvailableGenerators`.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
It doesn't seem to do anything.

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+///usr/bin/env node
```
Comment (SBoudrias) [inline lib/cli.js]: This is important though. It defines an executable in Unix.

**Comment by SBoudrias** [comment]
From a quick grepping through the code, this is used in `clear-config`, `home` and `update` routes.

**Comment by wtgtybhertgeghgtwtg** [comment]
There doesn't seem to be any tests that indicate that they aren't working.  Is the broken behavior not covered by a test?

**Comment by SBoudrias** [comment]
These are unit tests, so they're covered in isolation.

**Comment by wtgtybhertgeghgtwtg** [comment]
I apologize for the PR.

**Pull request closed without merging**